microprofile-lra/enable-microprofile-lra.cli

```diff
@@ -0,0 +1,8 @@
+batch
+/extension=org.wildfly.extension.microprofile.lra-coordinator:add
+/extension=org.wildfly.extension.microprofile.lra-participant:add
+/subsystem=microprofile-lra-coordinator:add
+/subsystem=microprofile-lra-participant:add
+run-batch
+
+reload
```

Comment:
please remove the reload, there is a known issue with server provisioning

Reply:
reload is required so this done manually by user is just unnecessary. It would be better to fix the issue.

Reply:
Ok, I rewrote it to manual stop/start as you have it in https://github.com/wildfly/quickstart/blob/main/ejb-security-context-propagation/README.adoc?plain=1#L119. But I still think this is bad user experience.